Fitness.xcodeproj/project.pbxproj

```diff
@@ -753,7 +793,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
-				DEVELOPMENT_TEAM = ZGMCXU7X3U;
+				DEVELOPMENT_TEAM = LL74343538;
```

Comment:
To what development team was this changed? Should be the Walker White team. If not we need to get you logged in with the account and get you the key.